etools-data-table-row.html

```diff
@@ -25,6 +25,11 @@
         background: var(--list-bg-color, #ffffff);
       }
 
+      :host([dark-bg-on-hover]) paper-material:hover {
+        background-color: var(--list-second-bg-color, #eeeeee);
```

Comment:
We could try to find a better name... :) ... `dark-bg-on-hover` doesn't fit if someone is using this and his `--list-second-bg-color` is defined in app-theme `#fff`... `secondary-bg-on-hover`? :D